Client/src/api/apiAuthentication.jsx

```diff
@@ -0,0 +1,8 @@
+import { LOCAL_SERVER, API_ADDRESS, QUERY_TYPE } from '../constants';
+import axios from 'axios';
+
+const apiAuthenctication = () => {
+  return axios.get(LOCAL_SERVER).then(result => result.data);
```

Comment:
1) LOCAL_SERVER_URL ?
2) you should better come up with generic name not mapping on the location of your server. It is local now, but may be not later.